index.js

```diff
@@ -8,6 +8,47 @@ const plugin = new Plugin({
   btpUri: 'btp+wss://interfaucet:' + process.env.TOKEN + '@amundsen.michielbdejong.com/api/17q3',
 })
 
+function getQuote(ippBuf) {
```

Comment:
Why don't you use the `ilp` module for quoting?

Reply:
I could copy this function into the `ilp` module and then include that dependency, but this is such a small function that I think it's better to keep it the way it is.